oidc/userinfo/oidc-userinfo-stub.ts

```diff
@@ -0,0 +1,20 @@
+import { UserInfo } from "./models";
+import { v4 as uuid } from "uuid";
+export interface Response {
```

Comment:
Line space between the imports, interface, newUserInfo and handler would make it a little easier to read